### PR TITLE
attachments image slider

### PR DESCRIPTION
**Pull request by juanfran**

### CODE REVIEW

File: app/modules/attachments/attachments.scss [hunk 1]
```diff
@@ -127,6 +127,28 @@
...
+            @include svg-size(3rem);
+            fill: $gray-light;
+        }
+        a:hover {
+            svg {
+                fill: $primary-light;
+                transition: fill .3s linear;
+            }
+        }
+    }
```
Comment (Xaviju) [inline app/modules/attachments/attachments.scss]: Can we do this?
```
svg {
    @include svg-size(3rem);
    fill: $gray-light;
      &:hover {
             fill: $primary-light;
             transition: fill .3s linear;
     }
 }
```

**Comment by Xaviju** [comment]
I can't find the e2e tests, should we add it in the PR?

**Comment by juanfran** [comment]
@Xaviju i've added the tests, thanks

**Comment by Xaviju** [comment]
Because of conflicts, I cannot merge this.

**Comment by superalex** [comment]
:dancers: :dancers: :dancers: :dancers: :dancers: :dancers:

**Comment by Xaviju** [comment]
(☞ﾟヮﾟ)☞

**Pull request merged**